5.0/en/0x03-Using-ASVS.md

```diff
@@ -1,72 +1,152 @@
 
-Requirements in this section are generally either defense in depth mechanisms or other useful but hard to implement controls. They may also relate to attacks which are a lot more niche or only relevant in certain circumstances.
+These requirements are generally critical or basic, first-layer of defense requirements for preventing common attacks that do not require other vulnerabilities or preconditions to be exploitable.
+
+In addition to the first layer of defense requirements, some requirements have less of an impact at higher levels, such as requirements related to passwords. Those are more important for Level 1, as from higher levels, the multi-factor authentication requirements become relevant.
+
+Level 1 is not necessarily penetration testable by an external tester without internal access to documentation or code (such as "black box" testing), although the lower number of requirements should make it easier to verify.
+
+### Level 2
+
+Most applications should be striving to achieve this level of security. Around 50% of the requirements in the ASVS are L2 meaning that an application needs to implement around 70% of the requirements in the ASVS in order to comply with L2.
+
+These requirements generally relate to either less common attacks or more complicated protections against common attacks. They may still be a first layer of defense, or they may require certain preconditions for the attack to be successful.
+
+### Level 3
+
+This level should be the goal for applications looking to demonstrate the highest levels of security and provides the final ~30% of requirements to comply with.
+
+Requirements in this section are generally either defense-in-depth mechanisms or other useful but hard-to-implement controls.
 
 ### Which level to achieve
 
-By moving to a priority based evaluation of each requirement, the levels become more of a reflection of the application security maturity of the organization and the application. Rather than the ASVS prescriptively stating what level an application should be at, an organization should decide what level it believes it should be at, depending on the sensitivity of the application and of course the expectations of the application's users.
+The priority-based levels are intended to provide a reflection of the application security maturity of the organization and the application. Rather than the ASVS prescriptively stating what level an application should be at, an organization should analyze its risks and decide what level it believes it should be at, depending on the sensitivity of the application and of course, the expectations of the application's users.
+
+For example, an early-stage startup that is only collecting limited sensitive data may decide that Level 1 is sufficient, but a bank may have difficulty justifying anything less than Level 3 to its customers for its online banking application.
```

Comment:
> early-stage startup that is only collecting limited sensitive data may decide that Level 1 is sufficient

I would say that this is questionnable (as everyone should aim for at least L2). I would better formulate it as:

> an early-stage startup may decide that is focuses of Level 1 for its initial security evaluation

Reply:
Ok I will make a change